packages/ketcher-react/src/script/ui/state/shared.ts

```diff
@@ -149,6 +150,14 @@ export function load(struct: Struct, options?) {
       const hasUnsupportedGroups = parsedStruct.sgroups.some(
         (sGroup) => !supportedSGroupTypes[sGroup.type],
       );
+      const hasMoleculeToMonomerConnections = parsedStruct.bonds.find(
```

Comment:
It's minor, but it seems that `.some` might fit better semantically here rather than `.find`Mi as you return some flag of whether any bond meets your condition, not the bond itself